### ISSUE TITLE
Solver segfaulting while running benchmarks

### ISSUE THREAD

**Issue by SaiCharanMarrivada** [meelgroup/ganak#29]
I tried to run your solver on dimacs files of ISCAS85 benchmarks, it segfaulted. Then, I tried on benchmarks in the benchmarks directory. The solver still segfaulted. I believe some change in your code base, left it in some vague state.

**Comment by msoos** [comment]
Right. Sorry. I’ll send you a binary today evening that works beautiful.
Sorry about that.

Mate

**Comment by SaiCharanMarrivada** [comment]
Thanks for your prompt response.

**Comment by msoos** [comment]
[ganak.gz](https://github.com/user-attachments/files/16354282/ganak.gz)

**Comment by msoos** [comment]
Please use that binary. You just need to `gunzip ganak.gz` and then `./ganak myfile.cnf`. Please do not use any preprocessor (B+E or Arjun) on the CNF before. Just run it with the original CNF. It has everything built-in and should work out-of-the box! Let me know how well it works,

Mate

**Comment by SaiCharanMarrivada** [comment]
Thanks

**Comment by SaiCharanMarrivada** [comment]
It works perfectly for all the benchmarks.

**Comment by msoos** [comment]
Hi! A new release has been made, please see here: https://github.com/meelgroup/ganak/releases It's even better :) I am closing because things should now work :)

Mate
